examples/api/test/material/dropdown_menu/dropdown_menu.0_test.dart

```diff
@@ -70,4 +71,40 @@ void main() {
     await tester.pumpAndSettle();
     expect(FocusScope.of(tester.element(iconMenu)).hasFocus, isTrue);
   });
+
+  testWidgets('DropdownMenu on small screen', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.DropdownMenuExample());
+
+    final Finder colorMenu = find.byType(DropdownMenu<example.ColorLabel>);
+    final Finder iconMenu = find.byType(DropdownMenu<example.IconLabel>);
+    expect(colorMenu, findsOneWidget);
+    expect(iconMenu, findsOneWidget);
+
+    Finder findMenuItem(String label) {
+      return find.widgetWithText(MenuItemButton, label).last;
+    }
+
+    await tester.tap(colorMenu);
+    await tester.pumpAndSettle();
+    final Finder menuBlue = findMenuItem('Blue');
+    await tester.ensureVisible(menuBlue);
+    await tester.tap(menuBlue);
+    await tester.pumpAndSettle();
+
+    await tester.tap(iconMenu);
+    await tester.pumpAndSettle();
+    final Finder menuSmile = findMenuItem('Smile');
+    await tester.ensureVisible(menuSmile);
+    await tester.tap(menuSmile);
+    await tester.pumpAndSettle();
+
+    expect(find.text('You selected a Blue Smile'), findsOneWidget);
+
+    // resize the screen to small screen and make sure no overflowed error appears
+    tester.binding.window.physicalSizeTestValue = const Size(200, 160);
+    tester.binding.window.devicePixelRatioTestValue = 1.0;
+    addTearDown(tester.binding.window.clearPhysicalSizeTestValue);
+
```

Comment:
```suggestion
    await tester.pump();
```
Otherwise the widget will not update after the size change and this test would still pass without the fix.
Or perhaps change the size of window at the start of the test?

Reply:
> Or perhaps change the size of window at the start of the test?

Actually, there are two overflowed errors: one at DropdownMenu row, one at the output result row `You selected a`. Thus, I needed to produce all error by selecting dropdown value too.
 I need to put the resize section afterward because it can't select value in the 2nd dropdown if the window is too small. 

Anyway, thank you for your suggestion about `tester.pump()`.